Extension/package.json

```diff
@@ -2761,6 +2761,17 @@
           "default": false,
           "markdownDescription": "%c_cpp.configuration.legacyCompilerArgsBehavior.markdownDescription%",
           "scope": "resource"
+        },
+        "C_Cpp.sshTargetsView": {
+          "type": "string",
+          "enum": [
+            "Enabled",
```

Comment:
Tarik is in the process of changing enum values to camelCase.  Can you change the values to `enabled`, `disabled`, `auto`?

Reply:
Sure